pynautobot/core/app.py

```diff
@@ -82,22 +82,73 @@ def choices(self):
 
         return self._choices
 
-    def custom_choices(self):
```

Comment:
Should we have `.custom_choices()` call `self.custom_fields()` so it is backwards compatible?

@jvanderaa @pszulczewski do we have any specific patterns of notifying the end user the method is deprecated?

Reply:
BTW, this is exactly the question that i had gotten, but i thought it depends on versioning. I guess if it's ok to wait new minor feature release (x.**_y_**.z) for this update, it can be as is. Otherwise i'm ready to upload a new commit, containing the original method and both of new, but renamed. Just let me know.

Reply:
I don't think we have a specific pattern today @joewesch. I think we should add in something for this.

Reply:
I think it would be good to add `.custom_choices()` to call `self.custom_fields()` as @joewesch suggested and we will keep it in 1.x release train and remove in 2.0.
We can also add `logging` and call `logger.warning()` that the method will be deprecated in 2.x.

Reply:
> I think it would be good to add `.custom_choices()` to call `self.custom_fields()` as @joewesch suggested and we will keep it in 1.x release train and remove in 2.0. We can also add `logging` and call `logger.warning()` that the method will be deprecated in 2.x.

Done